isic/ingest/admin.py

```diff
@@ -152,6 +158,36 @@ def failed_accessions(self, obj):
     def successful_accessions(self, obj):
         return intcomma(obj.successful_accessions_count)
 
+    @admin.action(description='Export file mapping')
+    @takes_instance_or_queryset
+    def export_file_mapping(self, request, queryset):
+        current_time = datetime.utcnow().strftime('%Y-%m-%d')
+        response = HttpResponse(content_type='text/csv')
+        response[
+            'Content-Disposition'
+        ] = f'attachment; filename="cohort_file_mapping_{current_time}.csv"'
+
+        writer = csv.DictWriter(
+            response, ['contributor', 'cohort', 'original_file_name', 'isic_id']
+        )
+
+        writer.writeheader()
+        for cohort in queryset.select_related('contributor').prefetch_related(
+            Prefetch('zips__accessions', queryset=Accession.objects.select_related('image'))
+        ):
+            for zip in cohort.zips.all():
+                for accession in zip.accessions.all():
+                    d = {
+                        'contributor': cohort.contributor.institution_name,
+                        'cohort': cohort.name,
+                        'original_file_name': accession.blob_name,
```

Comment:
Let's use the term `filename`, since that's what the metadata CSV column is called.